### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Youssef1313**
A point besides this PR, is there a specific coding style you follow ?

I'm seeing inconsistency in the following files:

https://github.com/nodatime/nodatime/blob/335de575f51912a45adf70fcef6a8a5dcd09595f/src/NodaTime.Benchmarks/NodaTimeTests/PeriodBenchmarks.cs#L11

https://github.com/nodatime/nodatime/blob/335de575f51912a45adf70fcef6a8a5dcd09595f/src/NodaTime.Benchmarks/NodaTimeTests/JsonNet/FormattingBenchmarks.cs#L15

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ Thanks for your interest in Noda Time. We appreciate all kinds of contributions,
 
 ## Basic Requirements
 
-If you want to contribute to the codebase, you're going to need a text editor or IDE. We recommend [Visual Studio Community 2017](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx) (Windows) or [Visual Studio Code](https://code.visualstudio.com/) (Windows/Linux/macOS).
+If you want to contribute to the codebase, you're going to need a text editor or IDE. We recommend [Visual Studio Community](https://visualstudio.microsoft.com/downloads/) (Windows), [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/), or [Visual Studio Code](https://code.visualstudio.com/) (Windows/Linux/macOS).
```
Comment (Youssef1313) [inline CONTRIBUTING.md]: Updated Visual Studio link and added link for Visual Studio for Mac.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ Thanks for your interest in Noda Time. We appreciate all kinds of contributions,
...
 
-You're also going to need .NET Core SDK installed - you can download it from [here.](https://www.microsoft.com/net/download/) (For Noda Time 3.0, we're using C# 8, which as of 2018-12-31 means using a preview of .NET Core.)
+You're also going to need .NET Core SDK installed - you can download it from [here](https://dotnet.microsoft.com/download).
```
Comment (Youssef1313) [inline CONTRIBUTING.md]: Updated the link. Removed the preview note. I think it's outdated now.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ git clone https://github.com/YOUR-USERNAME/nodatime.git
 
 Once you clone the repository, you'll have a [remote repository](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes) (or simply *remote*) called `origin`, that points to your forked repository on GitHub.
 
-You'll usually want to add another remote, pointing to the original repository on GitHub. It's an acepted convention to call this remote *upstream*. You can do it like this:
+You'll usually want to add another remote, pointing to the original repository on GitHub. It's an accepted convention to call this remote *upstream*. You can do it like this:
```
Comment (Youssef1313) [inline CONTRIBUTING.md]: Fix small typo.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jskeet) [reply CONTRIBUTING.md]: I'm not comfortable saying I *recommend* VS Code or VS for Mac for developing Noda Time, never having tried to do so myself. While they may be fine, I don't want to recommend something without experience. Something like "Visual Studio Code or Visual Studio for Mac should also work fine" would probably be better.

File: CONTRIBUTING.md (hunk 2 above)
Comment (jskeet) [reply CONTRIBUTING.md]: Yup, that's fine.

**Comment by jskeet** [comment]
In terms of conventions, yes, there will definitely be inconsistencies - hopefully limited to private members though. The capitalized form would normally be for constants, but it gets slightly grey. I'd generally say to try to follow the conventions of the code you're editing, or conventions for similar code if you're adding new.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Youssef1313) [reply CONTRIBUTING.md]: Hmm, I didn't get the point. Do you want the Visual Studio for Mac to be removed ?

Comment (jskeet) [reply CONTRIBUTING.md]: From the *recommendation* list, yes. You can mention it in a separate sentence along the lines of "Visual Studio for Mac and JetBrains Rider should be fine as well, but we haven't tried developing Noda Time using them" if you want.

**Pull request merged**